irohad/ametsuchi/impl/postgres_wsv_command.cpp

```diff
@@ -284,5 +306,23 @@ namespace iroha {
       }
       return true;
     }
+
+    bool PostgresWsvCommand::setAccountKV(const std::string &account_id,
+                                          const std::string &key,
+                                          const std::string &val) {
+      try {
+        transaction_.exec(
+            "UPDATE account\n"
+            "   SET data = jsonb_set(data,\'{"+key+"}\', \'\""+val+"\"\')"
```

Comment:
Looks totally insecure. Is there any other way?

Reply:
Yes, prepared statements look better.